components/builder-originsrv/src/data_store.rs

```diff
@@ -180,31 +176,23 @@ impl DataStore {
     ) -> SrvResult<originsrv::OriginProject> {
         let conn = self.pool.get(opc)?;
         let project = opc.get_project();
-        let token: Option<&str> = {
-            if project.has_vcs_auth_token() {
-                Some(project.get_vcs_auth_token())
-            } else {
-                None
-            }
-        };
-        let username: Option<&str> = {
-            if project.has_vcs_username() {
-                Some(project.get_vcs_username())
+        let install_id: Option<i64> = {
+            if project.has_vcs_installation_id() {
+                Some(project.get_vcs_installation_id() as i64)
             } else {
                 None
             }
         };
         let rows = conn.query(
-            "SELECT * FROM insert_origin_project_v2($1, $2, $3, $4, $5, $6, $7, $8)",
+            "SELECT * FROM insert_origin_project_v2($1, $2, $3, $4, $5, $6, $7)",
```

Comment:
We'll need to create a new stored procedure here (`_v3`). The function signature of a stored procedure needs to remain immutable

Reply:
since v2 has not been deployed to prod, do we need a v3?

Reply:
this could still break acceptance and development environments, no?

Reply:
Yeah, it'll still break acceptance unless we do some manual surgery on it to fix it. We can, and should, go through and remove old stored procedures. This will get cleaned up eventually when no other servers rely on it.